recipes/dm_control/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "dm_control" %}
+{% set version = "1.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dm_control-{{ version }}.tar.gz
+  sha256: 452f175bd04fdd0692c01cfa25dc09ff7ce61a81674f92cb1c4ac0f5cb6e1fb0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+
+requirements:
+  build:
+    - mujoco
+  host:
+    - python >=3.7
+    - mujoco
+    - pip
+    - pyparsing >=3.0.0
+
+  run:
+    - python >=3.7
+    - absl-py >=0.7.0
+    - dm-env
+    - dm-tree !=0.1.2
+    - glfw
+    - labmaze
+    - lxml
+    - mujoco >=2.3.0
+    - numpy >=1.9.0
+    - protobuf >=3.19.4
+    - pyopengl >=3.1.4
+    - pyparsing >=3.0.0
+    - requests
+    - setuptools !=50.0.0
+    - scipy
+    - tqdm
+    - pillow
+
+test:
+  imports:
+    - dm_control
+  commands:
+    - pip check
+    - pytest --pyargs dm_control -v -k "not (viewer or renderer or render)"
```

Comment:
pytest does very simple string matching on the serialization of the test here, e.g.
```
locomotion/tasks/corridors_test.py::CorridorsTest::test_termination_and_discount
```

"renderer" and "render" are redundant, and don't capture the entirety of tests that call into render code (i.e. the above example of a failing test contains neither "viewer" nor "render" in its serialization).

Unless dm_control has some pytest tags that can be used to skip various classes of issues, you're down to building a long chain of test names, e.g. [like so](https://github.com/conda-forge/scipy-feedstock/blob/main/recipe/meta.yaml#L72-L101).

Reply:
such long lines are not necessarily more readable but 🤷 

Glad you found a way to fix it. :)